src/App.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
-import './App.css';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 
-const logo = require('./logo.svg');
+import Header from './components/Header/Header';
+import Splash from './components/Splash/Splash';
 
 class App extends React.Component<{}, {}> {
   render() {
     return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.tsx</code> and save to reload.
-        </p>
+      <div>
+        <Header/>
```

Comment:
@SirSlothington Exactly what @NicksPatties said. It's basically a shorthand for not giving anything to the `Header` component. It's intentional in this case since the `Header` is stateless and simply renders a template.